client/my-sites/navigation/index.jsx

```diff
@@ -18,6 +20,15 @@ class MySitesNavigation extends React.Component {
 		event.stopPropagation();
 	};
 
+	renderSidebar() {
+		if ( 'jetpack-cloud' === config( 'sidebar' ) ) {
+			return (
+				<JetpackCloudSidebar path={ this.props.path } siteBasePath={ this.props.siteBasePath } />
```

Comment:
Seems like the logic here could just determine which component to use for the sidebar; and the props are the same. We could do:

```
const SidebarComponent = 'jetpack-cloud' === config( 'sidebar' ) ? JetpackCloudSidebar : Sidebar;
return <SidebarComponent path={ this.props.path } siteBasePath={ this.props.siteBasePath } />;
```